### DISCUSSION TITLE
Is it possible to vectorize text paths?

### DISCUSSION THREAD

**Discussion by Venom1991** [vvvv/SVG#1104] [Q&A] [answered]
This issue has nothing to do with any kind of a bug, not at all.  
I was wondering if it might be possible to emulate Aspose SVG's behavior regarding [text vectorization](https://docs.aspose.com/svg/net/text-vectorization/). Is this doable?

Thank you in advance.

**Comment by seanmcd27** [comment, marked as answer]
if you look at the discussion and issues history, this q been asked multiple times in various ways. the answer appears to be no in the official version.  if you look at my fork in the branch 'pointcollectiontyping' i have added a ToSvgPath() method to svgtextbase.cs that turns text into an svg path object for win32 only.  however, i've got it all entangled with other changes at the moment and i only have vs2019 so am working from tag 3.4.4 so i'm not prepared to submit a PR for this right now. but, maybe this will help
